### PR TITLE
Accessibility updates

### PR DESCRIPTION
**Pull request by phucduong86**
Details:
- Updated colors for better contrasts
- Update links styling for better consistency and visually discernable from other content type
- Implemented "visually-hidden" CSS class to be able to add off screen content for screen reader users (An example is the 2 "Visit website" links on the /books page will now read "Visit Learning Processing website" and "Visit Nature of Code website" by the screen reader.)
- Added focus style for improve accessibility for keyboard users.
- Turned a few capitalized text to normal text and use CSS to style them. This gives users with reading disabilities the ability to turn off CSS and read normal text. Reference read: https://github.com/humanmade/hm-pattern-library/issues/75

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -14,13 +14,13 @@ <h2>Latest videos</h2>
         <div class="line-charm"></div>
         <h2>Become a Patron</h2>
         <p>Daniel Shiffman is creating Creative Coding Video Tutorials</p>
-        <a href="https://www.patreon.com/codingtrain?ty=h" target="_blank" class="btn primary">BECOME A PATRON</a>
+        <a href="https://www.patreon.com/codingtrain?ty=h" target="_blank" class="btn primary">Become a patron</a>
```
Comment (shiffman) [inline index.html]:
```suggestion
        <a href="https://www.patreon.com/codingtrain?ty=h" target="_blank" class="btn primary">Become a Patron</a>
```

File: index.html [hunk 2]
```diff
@@ -14,13 +14,13 @@ <h2>Latest videos</h2>
         <div class="line-charm"></div>
         <h2>Become a Patron</h2>
         <p>Daniel Shiffman is creating Creative Coding Video Tutorials</p>
```
Comment (shiffman) [inline index.html]:
```suggestion
```

File: index.html [hunk 3]
```diff
@@ -14,13 +14,13 @@ <h2>Latest videos</h2>
...
-        <a href="https://www.patreon.com/codingtrain?ty=h" target="_blank" class="btn primary">BECOME A PATRON</a>
+        <a href="https://www.patreon.com/codingtrain?ty=h" target="_blank" class="btn primary">Become a patron</a>
     </div>
     <div class="more-videos content-wrapper article">
         <div class="line-charm"></div>
         <h2>See all videos on YouTube</h2>
         <p>All of Daniel's video courses can be found on YouTube</p>
```
Comment (shiffman) [inline index.html]:
```suggestion
        <p>For more videos:</p>
```

File: index.html [hunk 4]
```diff
@@ -14,13 +14,13 @@ <h2>Latest videos</h2>
...
-        <a href="https://youtube.com/user/shiffman" target="_blank" class="btn primary">SEE ALL COURSES ON YOUTUBE</a>
+        <a href="https://youtube.com/user/shiffman" target="_blank" class="btn primary">Ssee all courses on Youtube</a>
```
Comment (shiffman) [inline index.html]:
```suggestion
        <a href="https://www.youtube.com/thecodingtrain?sub_confirmation=1" target="_blank" class="btn primary">Subscribe to The Coding Train on YouTube.</a>
```

File: index.html [hunk 5]
```diff
@@ -46,7 +46,7 @@ <h3>Live stream updates</h3>
       <div class="form-group">
         <form action="//shiffman.us13.list-manage.com/subscribe/post?u=a3b0ca7250524fb6a4ddc769a&amp;id=a204c80fa5" method="post" id="mc-embedded-subscribe-form" name="mc-embedded-subscribe-form" class="validate" target="_blank" novalidate>
           <input id="user-email" type="email" value="" name="EMAIL" class="form" placeholder="Your email" aria-label="Your email"/>
-          <button type="submit" class="email-btn" name="subscribe">GET UPDATES</button>
+          <button type="submit" class="email-btn" name="subscribe">Get updates</button>
```
Comment (shiffman) [inline index.html]:
```suggestion
          <button type="submit" class="email-btn" name="subscribe">Get updates.</button>
```

File: index.html [hunk 6]
```diff
@@ -55,23 +55,23 @@ <h3>Live stream updates</h3>
     <div class="content">
       <h3 class="ct">The coding train</h3>
       <p>Videos on how to code</p>
-      <a href="http://thecodingtrain.com" target="_blank" class="primary" alt="The coding train logo">GO TO SITE</a>
+      <a href="http://thecodingtrain.com" target="_blank" class="primary" alt="The coding train logo">Go to <span class="visually-hidden">The Coding Train </span>site</a>
```
Comment (shiffman) [inline index.html]:
```suggestion
      <a href="http://thecodingtrain.com" target="_blank" class="primary" alt="The coding train logo">All aboard The Coding Train.</a>
```

File: index.html [hunk 7]
```diff
@@ -55,23 +55,23 @@ <h3>Live stream updates</h3>
...
       <p>Creative Coding for the Total Beginner</p>
-      <a href="https://www.amazon.com/Learning-Processing-Second-Edition-Programming/dp/0123944430/ref=as_li_ss_tl?ie=UTF8&linkCode=sl1&tag=learniproces-20&linkId=757eff7b49e61c381b17a43c7dec3eee" class="primary">GET THE BOOK</a>
+      <a href="https://www.amazon.com/Learning-Processing-Second-Edition-Programming/dp/0123944430/ref=as_li_ss_tl?ie=UTF8&linkCode=sl1&tag=learniproces-20&linkId=757eff7b49e61c381b17a43c7dec3eee" class="primary">Get the <span class="visually-hidden">Learning Processing </span>book</a>
     </div>
     <img class="book" src="images/book-lp.png" alt="Learn Processing book" width="131px"/>
   </div>
   <div class="feature noc">
     <div class="content">
-      <h3>NATURE OF CODE</h3>
+      <h3>Nature of code</h3>
```
Comment (shiffman) [inline index.html]:
```suggestion
      <h3>Nature of Code</h3>
```

File: index.html [hunk 8]
```diff
@@ -55,23 +55,23 @@ <h3>Live stream updates</h3>
...
+      <a href="http://thecodingtrain.com" target="_blank" class="primary" alt="The coding train logo">Go to <span class="visually-hidden">The Coding Train </span>site</a>
     </div>
      <img class="rainbow" src="images/train.png" alt="The Coding Train Logo" width="300px"/>
   </div>
   <div class="feature lp">
     <div class="content">
       <h3>Learning Processing</h3>
       <p>Creative Coding for the Total Beginner</p>
-      <a href="https://www.amazon.com/Learning-Processing-Second-Edition-Programming/dp/0123944430/ref=as_li_ss_tl?ie=UTF8&linkCode=sl1&tag=learniproces-20&linkId=757eff7b49e61c381b17a43c7dec3eee" class="primary">GET THE BOOK</a>
+      <a href="https://www.amazon.com/Learning-Processing-Second-Edition-Programming/dp/0123944430/ref=as_li_ss_tl?ie=UTF8&linkCode=sl1&tag=learniproces-20&linkId=757eff7b49e61c381b17a43c7dec3eee" class="primary">Get the <span class="visually-hidden">Learning Processing </span>book</a>
```
Comment (shiffman) [inline index.html]:
```suggestion
      <a href="https://www.amazon.com/Learning-Processing-Second-Edition-Programming/dp/0123944430/ref=as_li_ss_tl?ie=UTF8&linkCode=sl1&tag=learniproces-20&linkId=757eff7b49e61c381b17a43c7dec3eee" class="primary">Get the <span class="visually-hidden">Learning Processing </span>book.</a>
```

File: index.html [hunk 9]
```diff
@@ -55,23 +55,23 @@ <h3>Live stream updates</h3>
...
     </div>
     <img class="book" src="images/book-lp.png" alt="Learn Processing book" width="131px"/>
   </div>
   <div class="feature noc">
     <div class="content">
-      <h3>NATURE OF CODE</h3>
+      <h3>Nature of code</h3>
       <p>Simulations of Natural Systems</p>
-      <a href="https://www.amazon.com/gp/product/0985930802/ref=as_li_tf_tl?ie=UTF8&camp=1789&creative=9325&creativeASIN=0985930802&linkCode=as2&tag=natureofcode-20" target="_blank" class="primary">GET THE BOOK</a>
+      <a href="https://www.amazon.com/gp/product/0985930802/ref=as_li_tf_tl?ie=UTF8&camp=1789&creative=9325&creativeASIN=0985930802&linkCode=as2&tag=natureofcode-20" target="_blank" class="primary">Get the <span class="visually-hidden">Nature of code </span>book</a>
```
Comment (shiffman) [inline index.html]:
```suggestion
      <a href="https://www.amazon.com/gp/product/0985930802/ref=as_li_tf_tl?ie=UTF8&camp=1789&creative=9325&creativeASIN=0985930802&linkCode=as2&tag=natureofcode-20" target="_blank" class="primary">Get the <span class="visually-hidden">Nature of Code </span>book.</a>
```

File: learning.html [hunk 10]
```diff
@@ -62,7 +62,7 @@ <h3>COURSES</h3>
     </div>
 
     <br>
-    <h3>OPEN SOURCE PROJECTS</h3>
+    <h3>Open source projects</h3>
```
Comment (shiffman) [inline learning.html]:
```suggestion
    <h3>Open Source Projects</h3>
```

File: learning.html [hunk 11]
```diff
@@ -77,14 +77,14 @@ <h3>OPEN SOURCE PROJECTS</h3>
     </div>
 
     <br>
-    <h3>OLD BLOG POSTS</h3>
+    <h3>Old blog posts</h3>
```
Comment (shiffman) [inline learning.html]:
```suggestion
    <h3>Retired Blog</h3>
```

File: learning.html [hunk 12]
```diff
@@ -77,14 +77,14 @@ <h3>OPEN SOURCE PROJECTS</h3>
...
   	{% for post in site.posts limit:25 %}
   	  <div class="quick-link">
   	      <a href="{{ post.url }}" class="secondary">{{ post.title }}</a>
   	  </div>
     {% endfor %}
     <div class="quick-link">
-      <a href="/blog.html" class="secondary">SEE ALL</a>
+      <a href="/blog.html" class="secondary">See all</a>
```
Comment (shiffman) [inline learning.html]:
```suggestion
      <a href="/blog.html" class="secondary">See all.</a>
```

**Comment by phucduong86** [comment]
Hi Dan,

Here's some screenshots of the main elements. I tried not to change the color scheme too much and just darken the existing colors. The biggest change is the yellow color - it's looking more like a green now the other option is  keeping the green but make the text darker. I feel that the later would break the consistency of the color scheme on the right panel.
image
image
image
image
image

**Comment by shiffman** [comment]
Thank you @phucduong86, this is so great! I think the green color probably needs re-thinking but accessibility for me is a higher priority than looking nice. So perhaps we merge this but open a new issue inviting proposals for different color schemes with the contrast requirements?

**Comment by phucduong86** [comment]
> Thank you @phucduong86, this is so great! I think the green color probably needs re-thinking but accessibility for me is a higher priority than looking nice. So perhaps we merge this but open a new issue inviting proposals for different color schemes with the contrast requirements?

That sounds great Dan! Thank you for the review.

**Pull request merged**